components/integrations/confirm_integration/confirm_integration.test.jsx

```diff
@@ -51,6 +55,17 @@ describe('components/integrations/ConfirmIntegration', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
+    test('should match callback URLs of OAuth Apps', () => {
+        props.location.search = getSearchString('oauth2-apps');
+        const {container} = renderWithIntl(
+            <Router history={browserHistory}>
+                <ConfirmIntegration {...props}/>
+            </Router>
+        );
+
+        expect(container.querySelector('.word-break--all')).toHaveTextContent('URL(s): https://someCallback, https://anotherCallback');
```

Comment:
Like this could be something like 
```javascript
const wrapper = mountWithIntl(...);
expect(wrapper.find('.word-break--all').text()).toBe('URL(s): https://someCallback, https://anotherCallback');
```

Reply:
Yeah, I agree with this one.  I started with a different test case and just ended up verifying rendered text. I'll change once we have an agreement whether we can use testing-library or not.